internal/logout/command_test.go

```diff
@@ -197,8 +216,34 @@ func newLoginCmd(auth *ccloudv1mock.Auth, userInterface *ccloudv1mock.UserInterf
 	return loginCmd, cfg
 }
 
-func newLogoutCmd(cfg *config.Config, netrcHandler netrc.NetrcHandler) (*cobra.Command, *config.Config) {
-	logoutCmd := New(cfg, climock.NewPreRunnerMock(nil, nil, nil, nil, cfg), netrcHandler)
+func newLogoutCmd(auth *ccloudv1mock.Auth, userInterface *ccloudv1mock.UserInterface, isCloud bool, req *require.Assertions, netrcHandler netrc.NetrcHandler, authTokenHandler pauth.AuthTokenHandler, contextName string) (*cobra.Command, *config.Config) {
+	config.SetTempHomeDir()
+	cfg := config.AuthenticatedConfigMockWithContextName(contextName)
+	var mdsClient *mdsv1.APIClient
+	if !isCloud {
```

Comment:
This looks like duplicate code. Can we de-duplicate?

Reply:
Is there a place in the code base where shared unit test helpers already exist? That way we can just have one `newLoginCmd` and `newLogoutCmd` too.

Reply:
Not yet, but since these tests span two packages, I'd expect it to go somewhere in pkg/

Reply:
I don't think I can put them in `pkg` because of import cycles, but I think the best place to put them would be `mock`.

Reply:
I tried putting this in existing packags that made sense in `pkg/` (`auth` and `mock`) but both result in circular imports, so I had to create a new directory unfortunately.